system/config.php

```diff
@@ -49,5 +49,7 @@
 	'sendmail' => 'SwiftMailerTransport'
 ));
 
+Config::set('system.gc_maxlifetime', 21600);
```

Comment:
This number doesn't fall into your lifetime bounds check in Web, 21600 seconds is 6 hours, but your lower bounds is a day